benchmarks/benchmarks/stats.py

```diff
@@ -174,3 +174,24 @@ def time_binned_statistic_dd_reuse_bin(self, statistic):
         stats.binned_statistic_dd(
             [self.inp[0], self.inp[1]], self.inp[2], statistic=statistic,
             binned_statistic_result=self.ret)
+
+
+class ContinuousFitAnalyticalMLEOverride(Benchmark):
+    pretty_name = "Fit Methods Overriden with Analytical MLEs"
+    param_names = ['distribution name']
+    params = ["pareto", "laplace"]
+
+    distributions = {"pareto": {"self": stats.pareto, "shapes": {"b": 2},
```

Comment:
Does this allow you to do multiple tests with the same distribution? For example, you're going to get pretty different times for "rayleigh" depending on which parameter is fixed and if neither is fixed.

Reply:
Yes, you could define multiple `time_` methods and then in the interface you can select which to view later. So, there could be 
- time_all_free
- time_loc_fixed
- time_scale_fixed
- time_shape_fixed